nvflare/tool/job/config/configer.py

```diff
@@ -336,6 +374,17 @@ def build_config_file_indices(job_folder: str, app_names: List[str]) -> Dict[str
                     config_files.append(file)
                     app_config_files[app_name] = config_files
 
+    for app_name in app_names:
+        custom_dir = os.path.join(job_folder, app_name, "custom")
+        for root, dirs, files in os.walk(custom_dir):
+            for f in files:
+                for ext in config_extensions:
+                    if f.endswith(ext):
+                        file = os.path.join(root, f)
+                        config_files = app_config_files.get(app_name, [])
+                        config_files.append(file)
+                        app_config_files[app_name] = config_files
```

Comment:
What if the custom folder inside it is not flat?
Means there are also folders inside the custom folder.

For example: "custom/cool_dir/something.conf", do we want to support this?
Or we could just say this is not supported.